.env.example

```
@@ -662,3 +662,18 @@ OLLAMA_API_BASE=http://localhost:11434 # Alternative to OLLAMA_BASE_URL
 # =============================================================================
 NEUROLINK_PACKAGE=@juspay/neurolink    # Package identifier
 PROMPT_TYPE=default                    # Prompt template type
+
+# =============================================================================
+# OBSERVABILITY & ANALYTICS
+# =============================================================================
+# Langfuse Configuration
+# Sign up at: https://cloud.langfuse.com or self-host
+LANGFUSE_PUBLIC_KEY=pk-lf-your-public-key-here
+LANGFUSE_SECRET_KEY=sk-lf-your-secret-key-here
+LANGFUSE_BASE_URL=https://cloud.langfuse.com
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The LANGFUSE_BASE_URL key should go before the LANGFUSE_PUBLIC_KEY key

(UnorderedKey)